src/pseudopeople/utilities.py

```diff
@@ -192,6 +192,35 @@ def cleanse_integer_columns(column: pd.Series) -> pd.Series:
     return column
 
 
+def number_of_tokens_per_string(s1, s2):
+    """
+    Calculates the number of tokens in each string of a series.
+    s1 is a pd.Series of tokens and we want to see how many tokens exist in each
+    string of s2.
+    """
+
+    s2 = s2.astype(str)
+    strings = s2.unique()
```

Comment:
I'm not able to fully follow the logic here, but it seems like `.unique()` would return a misaligned index?

Reply:
Yes that is why we map it back at the end of the function.